.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,28 @@
+name: Reviewdog
+on: [push, pull_request]
```

Comment:
`push` event doesn't work with github-pr-review reporter, can you remove it?

Reply:
fixed: 5f26ce8